databuilder/dsl.py

```diff
@@ -46,26 +46,8 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from datetime import datetime
-from typing import Generic, TypeVar, overload
-
-from .query_model_old import (
-    BaseTable,
-    Codelist,
-    Comparator,
-    DateAddition,
-    DateDeltaAddition,
-    DateDeltaSubtraction,
-    DateDifference,
-    DateSubtraction,
-    RoundToFirstOfMonth,
-    RoundToFirstOfYear,
-    Row,
-    Table,
-    Value,
-    ValueFromAggregate,
-    ValueFromCategory,
-)
+
+from .query_model_old import BaseTable, Codelist, Comparator, Value, ValueFromAggregate
```

Comment:
I'm sure this is fine for now but, just for my own understanding, why is a tiny bit of the old DSL staying and using the old query model?